style.css

```diff
@@ -27,9 +27,9 @@ nav {
   display: block;
   width: 25px;
   height: 3px;
-  margin: 5px 20px;
+  margin: 5px 30px;
```

Comment:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇